src/detect-engine-content-inspection.c

```diff
@@ -59,6 +59,8 @@
 #include "util-unittest-helper.h"
 #include "util-profiling.h"
 
+#include "rust-bindings.h"
```

Comment:
@jasonish would we include "rust.h" or "rust-bindings.h" ?

Reply:
Looks like `rust.h` is the preferred way.

Reply:
"rust-bindings.h" is used in some files, and I am not sure I know the difference...